package.json

```diff
@@ -148,13 +147,14 @@
       "@walletconnect/client@1.7.8": "patches/@walletconnect__client@1.7.8.patch",
       "reflect-metadata@0.1.13": "patches/reflect-metadata@0.1.13.patch",
       "bloom-filters@3.0.0": "patches/bloom-filters@3.0.0.patch",
-      "web3-core@1.8.0": "patches/web3-core@1.8.0.patch",
       "urlcat@3.1.0": "patches/urlcat@3.1.0.patch",
       "@chainsafe/as-sha256@0.3.1": "patches/@chainsafe__as-sha256@0.3.1.patch",
       "@protobufjs/inquire@1.1.0": "patches/@protobufjs__inquire@1.1.0.patch",
       "@splinetool/runtime@0.9.342": "patches/@splinetool__runtime@0.9.342.patch",
       "guess-calling-code@1.1.7": "patches/guess-calling-code@1.1.7.patch",
-      "@walletconnect/socket-transport@1.7.8": "patches/@walletconnect__socket-transport@1.7.8.patch"
+      "@walletconnect/socket-transport@1.7.8": "patches/@walletconnect__socket-transport@1.7.8.patch",
+      "web3-core@1.10.2": "patches/web3-core@1.10.2.patch",
+      "web3@1.10.2": "patches/web3@1.10.2.patch"
```

Comment:
web3 already has 4.1.2, do we need to stick on v1? cc @guanbinrui

Reply:
I'm afraid we should. They maintain completely different APIs.